source/OutfitterPanel.cpp

```diff
@@ -538,6 +524,23 @@ ShopPanel::TransactionResult OutfitterPanel::CanSellOrUninstall(const string &ve
 	if(!playerShip)
 		return "No ship selected.";
 
+	if(verb == "sell")
```

Comment:
I pulled in all the logic that was external to this function so that instead of having three functions wrapping this and calling it, the logic is all within and this is the only function that will be called in these contexts instead of CanSell, CanUninstall, and CanMoveToStorage -- which have all been removed.